packages/typedoc-plugin-appium/lib/converter/converter.ts

```diff
@@ -0,0 +1,417 @@
+    refl?: DeclarationReflectionWithReflectedType
+  ): string[] {
+    if (refl) {
+      const props = findChildByNameAndGuard(refl, propName, isParamsArray);
+      const names = props?.type.target.elements.map((el: LiteralType) => String(el.value)) ?? [];
+      return names.filter((name) => {
+        if (!name) {
+          this.#log.warn('Found empty %s parameter', propName);
+          return false;
+        }
+        return true;
+      });
+    }
+    return [];
+  }
+
+  /**
+   * Gathers info about an `executeMethodMap` prop in a driver
+   * @param refl A class which may contain an `executeMethodMap` static property
+   * @returns List of "execute commands", if any
+   */
+  #convertExecuteMethodMap(refl: DeclarationReflectionWithReflectedType): ExecCommandDataSet {
+    const executeMethodMap = findChildByNameAndGuard(
+      refl,
+      NAME_EXECUTE_METHOD_MAP,
+      isExecMethodDefReflection
+    );
+    const commandRefs: ExecCommandDataSet = new Set();
+    if (!executeMethodMap) {
+      // no execute commands in this class
+      return commandRefs;
+    }
+
+    const newMethodProps = filterChildrenByKind(executeMethodMap, ReflectionKind.Property);
+    for (const newMethodProp of newMethodProps) {
+      const {comment, originalName: script} = newMethodProp;
+
+      const commandProp = findChildByNameAndGuard(
+        newMethodProp,
+        NAME_COMMAND,
+        isCommandPropDeclarationReflection
+      );
+
+      if (!commandProp) {
+        // this is unusual
+        this.#log.warn(
+          'Execute method map in %s has no "command" property for %s',
+          refl.name,
+          script
+        );
+        continue;
+      }
+
+      if (!_.isString(commandProp.type.value) || _.isEmpty(commandProp.type.value)) {
+        this.#log.warn(
+          'Execute method map in %s has an empty or invalid "command" property for %s',
+          refl.name,
+          script
+        );
+        continue;
+      }
+      const command = String(commandProp.type.value);
+
+      const paramsProp = findChildByNameAndGuard(
+        newMethodProp,
+        NAME_PARAMS,
+        isReflectionWithReflectedType
+      );
+      const requiredParams = this.#convertRequiredCommandParams(paramsProp);
+      const optionalParams = this.#convertOptionalCommandParams(paramsProp);
+      commandRefs.add({
+        command,
+        requiredParams,
+        optionalParams,
+        script,
+        comment,
+      });
+    }
+    return commandRefs;
+  }
+
+  /**
+   * Extracts information about `MethodMap` objects
+   * @param refl - Some reflection we want to inspect. Could refer to a module or a class
+   * @returns Lookup of routes to {@linkcode CommandMap} objects
+   */
+  #convertMethodMap(refl: DeclarationReflection): RouteMap {
+    const routes: RouteMap = new Map();
+
+    const methodMap = isBaseDriverDeclarationReflection(refl)
+      ? refl.getChildByName(NAME_METHOD_MAP)
```

Comment:
applying ternary operator to refl.getChildByName argument might save some code lines

Reply:
```methodMap =  refl.getChildByName(isBaseDriverDeclarationReflection(refl) ? NAME_METHOD_MAP : ...)```